Bottom: 2a922d84847798f6a5d78ca37a88166a95661afb
Top:    800cf7d46d4eb190e0188ceebc7070fbeb1daa8c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:26:31 +0000

Refresh of polygon1-c-remove-now-unused

---

diff --git a/src/polygon1.c b/src/polygon1.c
index bbe3f6e..dcdda21 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -105,10 +105,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #define DEBUG_LABEL
 #define DEBUG_ALL_LABELS
 #define DEBUG_JUMP
